helm/pmm-server/templates/statefulset.yaml

```diff
@@ -21,6 +28,13 @@ spec:
       labels:
         app: {{ template "percona.fullname" . }}
         component: pmm
+{{- if .Values.statefulSet.podTemplate.labels }}
+{{ toYaml .Values.statefulSet.podTemplate.labels | indent 8 }}
+{{- end }}
+{{- with .Values.statefulSet.podTemplate.annotations }}
```

Comment:
Hi @andrewkro ! Sorry, but this got sidetracked somehow!
Could you please rebase on current main branch and the annotations for the pod will need to be changed a bit since we set annotations in case if platform is openshift so we need to merge them and not that one overrides the other values.
Thanks!